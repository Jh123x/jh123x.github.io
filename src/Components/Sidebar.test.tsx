import * as React from "react";
import { describe, expect, test } from "@jest/globals";
import Sidebar from "./Sidebar";

describe("Sidebar", () => {
  test("should render correctly toggled", () => {
    expect(<Sidebar navToggle={true} />).toMatchSnapshot();
  });
  test("should render correctly untoggled", () => {
    expect(<Sidebar navToggle={false} />).toMatchSnapshot();
  });
});
