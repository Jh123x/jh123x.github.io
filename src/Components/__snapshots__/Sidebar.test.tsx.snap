// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sidebar should render correctly toggled 1`] = `
<Sidebar
  navToggle={true}
/>
`;

exports[`Sidebar should render correctly untoggled 1`] = `
<Sidebar
  navToggle={false}
/>
`;
